api/v1/db/oracledb/developers-dao.js

```diff
@@ -0,0 +1,60 @@
+const appRoot = require('app-root-path');
+const config = require('config');
+const _ = require('lodash');
+
+const { serializeDevelopers, serializeDeveloper } = require('../../serializers/developers-serializer');
+
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+
+const { endpointUri } = config.get('server');
+
+/**
+ * @summary Return a list of developers
+ * @function
+ * @returns {Promise<Object[]>} Promise object represents a list of developers
+ */
+const getDevelopers = async () => {
+  const connection = await conn.getConnection();
+  const sqlQuery = 'SELECT ID, NAME, WEBSITE FROM DEVELOPERS';
+  try {
+    const rawDevelopersReponse = await connection.execute(sqlQuery);
+    const rawDevelopers = rawDevelopersReponse.rows;
+    const serializedDevelopers = serializeDevelopers(rawDevelopers, endpointUri);
```

Comment:
These two lines can be combined with using the [JS object destructing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring):

```js
const { rows } = await connection.execute(sqlQuery);
const serializedDevelopers = serializeDevelopers(rows, endpointUri);
```

Example: https://github.com/osu-mist/terms-api/blob/master/api/v1/db/oracledb/terms-dao.js#L37

Reply:
Combined the two lines